iceoryx_posh/test/integrationtests/service_discovery/test_roudi_findservice.cpp

```diff
@@ -205,13 +205,13 @@ TEST_F(RoudiFindService_test, InterfacePort)
     auto interfacePortData = receiverRuntime->getMiddlewareInterface(iox::capro::Interfaces::SOMEIP);
     iox::popo::InterfacePort interfacePort(interfacePortData);
 
-    iox::capro::CaproMessage caproMessage;
     this->InterOpWait();
 
     bool serviceFound = false;
 
-    while (interfacePort.getCaProMessage(caproMessage))
+    while (auto maybeCaProMessage = interfacePort.getCaProMessage())
```

Comment:
`maybe` sounds here a bit wrong, but I also don't have a better name

Reply:
I have the feeling we somehow agreed that maybe* is a good name for the optional return